templates/people/edit.html

```diff
@@ -12,5 +12,12 @@ <h1>{{ user }}{% if role %} <small>{{ role }}</small>{% endif %}</h1>
 		{{ form | crispy }}
 		<input type="submit" class="btn btn-primary" value="Save profile" />
 	</form>
+
+	<br>
+	<form method="POST" action="{% url 'people:profile_delete' fair.year pk %}">
+	{% csrf_token %}
+	<input type="submit" class="btn btn-primary" value="Delete profile" onclick="return confirm('Are you sure that you want to delete your profile? This means that all data will be erased from Armada and cannot be restored. If you want to continue using Armada in the future it means that you will have to create a new profile.')" />
```

Comment:
Can you make that the `onClick` text is passed from the `people/views.py` as a variable ?
Just make sure that the text is a variable that's easier for the next-developer to change it.